sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/VaultList.cs

```diff
@@ -1,16 +1,18 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-// <auto-generated/>
-
 #nullable disable
 
+using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using Azure.Core;
 
 namespace Azure.ResourceManager.NetApp.Models
 {
     /// <summary> List of Vaults. </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Since this class is internal, our customer will never see it and we do not need this change